docs/serving/installing-cert-manager.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "Performing a Cert-Manager Installation"
+weight: 15
+type: "docs"
+---
+
+Within Knative, we provide a feature of automatically provisioning and 
+configuring TLS certificates to terminate the external TLS connection. This 
+feature needs [Cert-Manager](https://github.com/jetstack/cert-manager) to be 
+installed within your cluster.
+This doc provides a guide to install Cert-Manager.
+
+<!-- TODO(zhiminx) add the link of networking-certmanager deployment after the code is checked in.-->
+> Note: Cert-Manager CRDs need to be installed if Knative 
+> [networking-certmanager deployment]() is installed in your 
+> cluster (no matter whether the auto TLS will be used). 
```

Comment:
This means that this should be linked from ALL of our install guides as well.  I would include the note that `cert-manager` can be excluded by adding `-l networking.knative.dev/certificate-provider!=cert-manager` to the `kubectl apply`.

Reply:
Updated most of docs in the `install` directory. Added TODO for openshift and minishift as they are using `oc`. Need experts to add the instructions there.

Reply:
cc @bbrowning @markusthoemmes